doc/sphinxdoc/models.rst

```diff
@@ -0,0 +1,653 @@
+aircraft, airplane`, `Bicycle`, `Skateboard`, `Engine`, `Light engine (high
+frequency)`, `Dental drill, dentist's drill`, `Lawn mower`, `Chainsaw`, `Medium
+engine (mid frequency)`, `Heavy engine (low frequency)`, `Engine knocking`,
+`Engine starting`, `Idling`, `Accelerating, revving, vroom`, `Door`, `Doorbell`,
+`Ding-dong`, `Sliding door`, `Slam`, `Knock`, `Tap`, `Squeak`, `Cupboard open or
+close`, `Drawer open or close`, `Dishes, pots, and pans`, `Cutlery, silverware`,
+`Chopping (food)`, `Frying (food)`, `Microwave oven`, `Blender`, `Water tap,
+faucet`, `Sink (filling or washing)`, `Bathtub (filling or washing)`, `Hair
+dryer`, `Toilet flush`, `Toothbrush`, `Electric toothbrush`, `Vacuum cleaner`,
+`Zipper (clothing)`, `Keys jangling`, `Coin (dropping)`, `Scissors`, `Electric
+shaver, electric razor`, `Shuffling cards`, `Typing`, `Typewriter`, `Computer
+keyboard`, `Writing`, `Alarm`, `Telephone`, `Telephone bell ringing`,
+`Ringtone`, `Telephone dialing, DTMF`, `Dial tone`, `Busy signal`, `Alarm
+clock`, `Siren`, `Civil defense siren`, `Buzzer`, `Smoke detector, smoke alarm`,
+`Fire alarm`, `Foghorn`, `Whistle`, `Steam whistle`, `Mechanisms`, `Ratchet,
+pawl`, `Clock`, `Tick`, `Tick-tock`, `Gears`, `Pulleys`, `Sewing machine`,
+`Mechanical fan`, `Air conditioning`, `Cash register`, `Printer`, `Camera`,
+`Single-lens reflex camera`, `Tools`, `Hammer`, `Jackhammer`, `Sawing`, `Filing
+(rasp)`, `Sanding`, `Power tool`, `Drill`, `Explosion`, `Gunshot, gunfire`,
+`Machine gun`, `Fusillade`, `Artillery fire`, `Cap gun`, `Fireworks`,
+`Firecracker`, `Burst, pop`, `Eruption`, `Boom`, `Wood`, `Chop`, `Splinter`,
+`Crack`, `Glass`, `Chink, clink`, `Shatter`, `Liquid`, `Splash, splatter`,
+`Slosh`, `Squish`, `Drip`, `Pour`, `Trickle, dribble`, `Gush`, `Fill (with
+liquid)`, `Spray`, `Pump (liquid)`, `Stir`, `Boiling`, `Sonar`, `Arrow`,
+`Whoosh, swoosh, swish`, `Thump, thud`, `Thunk`, `Electronic tuner`, `Effects
+unit`, `Chorus effect`, `Basketball bounce`, `Bang`, `Slap, smack`, `Whack,
+thwack`, `Smash, crash`, `Breaking`, `Bouncing`, `Whip`, `Flap`, `Scratch`,
+`Scrape`, `Rub`, `Roll`, `Crushing`, `Crumpling, crinkling`, `Tearing`, `Beep,
+bleep`, `Ping`, `Ding`, `Clang`, `Squeal`, `Creak`, `Rustle`, `Whir`, `Clatter`,
+`Sizzle`, `Clicking`, `Clickety-clack`, `Rumble`, `Plop`, `Jingle, tinkle`,
+`Hum`, `Zing`, `Boing`, `Crunch`, `Silence`, `Sine wave`, `Harmonic`, `Chirp
+tone`, `Sound effect`, `Pulse`, `Inside, small room`, `Inside, large room or
+hall`, `Inside, public space`, `Outside, urban or manmade`, `Outside, rural or
+natural`, `Reverberation`, `Echo`, `Noise`, `Environmental noise`, `Static`,
+`Mains hum`, `Distortion`, `Sidetone`, `Cacophony`, `White noise`, `Pink noise`,
+`Throbbing`, `Vibration`, `Television`, `Radio`, `Field recording`
+
+Dataset: AudioSet.
+
+Output: activations.
+
+This model is useful for audio embeddings.
+
+Models:
+
+* ``audioset-yamnet``
```

Comment:
Hyperlink to the latest version of the model instead of the names?

Reply:
Maybe some does not necessarily want the last version but access to all version? Also (in general, not related to this model) we have separate files for TensorFlow.js so this won't work well for all use-cases.